<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneEditor</class>
 <widget class="QMainWindow" name="SceneEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hair - Scene Editor</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="controlsBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>182</width>
      <height>670</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>182</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>182</width>
      <height>800</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>1008</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="testHolder">
      <item>
       <widget class="QPushButton" name="showHideGroupMaps">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Maps</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupMaps">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="densityMapOption">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Length Map</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="directionMapOption">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Direction Map</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showHideGroupGlobals">
        <property name="maximumSize">
         <size>
          <width>74</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Globals</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupGlobals">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="inputGlobalDensity">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Hair Density</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderGlobalDensity">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputGlobalLength">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderGlobalLength">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Hair Length</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearWhiteButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>440</y>
           <width>171</width>
           <height>32</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Clear White</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearBlackButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>410</y>
           <width>171</width>
           <height>32</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Clear Black</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showHideGroupBrush">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Brush</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBrush">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QSlider" name="sliderBrushGrayscale">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>2550</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputRadius">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputBrushGrayscale">
         <property name="geometry">
          <rect>
           <x>109</x>
           <y>220</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelBrushGrayscale">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Color (gray)</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderRadius">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputOpacity">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderOpacity">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>2550</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>151</width>
           <height>101</height>
          </rect>
         </property>
         <property name="title">
          <string>Brush Falloff</string>
         </property>
         <widget class="QRadioButton" name="falloffLinear">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Linear</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="falloffConstant">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Constant</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="falloffQuadratic">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Quadratic</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBrushDirection">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>151</width>
           <height>121</height>
          </rect>
         </property>
         <property name="title">
          <string>Brush Direction</string>
         </property>
         <widget class="QRadioButton" name="brushDirRight">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Right</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="brushDirLeft">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="brushDirUp">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="brushDirDown">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>102</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>182</width>
       <height>461</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>182</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>600</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>178</width>
        <height>556</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="applyAndCloseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Apply and Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>505</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>505</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
